apps/cookbook/src/app/examples/form-field-example/form-field-example.component.html

```diff
@@ -1,25 +1,33 @@
 <kirby-page title="Form Field">
   <kirby-page-content>
+    <cookbook-form-field-example-configuration
+      [size]="size"
+      (sizeChanged)="onSizeChange($event)"
```

Comment:
Consider using banana-in-a-box syntax:
```suggestion
      [(size)]="size"
```

Reply:
Done